eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/README.md

```diff
@@ -180,15 +172,7 @@ You can debug your sample by adding the saved output values to the tool's enviro
 1.  Verify in your app’s logs that similar messages were posted:
 
 ```shell
-New message received: 'Hello world, 17' ...
-Message 'Hello world, 17' successfully checkpointed
-...
-New message received: 'Hello world, 18' ...
-Message 'Hello world, 18' successfully checkpointed
-...
-New message received: 'Hello world, 27' ...
-Message 'Hello world, 27' successfully checkpointed
-
+Received message: Test event
```

Comment:
nit: How about adding all related logs in the source code? For example: `Sending message using...` and `Sent message to Event Hub.`